### PR TITLE
Issue #1116: BigQuery write fails with MessageSize is too large

### PR DESCRIPTION
**Pull request by agrawal-siddharth**

### CODE REVIEW

File: bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryDirectDataWriterHelper.java [hunk 1]
```diff
@@ -176,8 +176,10 @@ private StreamWriter createStreamWriter(String writeStreamName) {
    */
   public void addRow(ByteString message) throws IOException {
     int messageSize = message.size();
```
Comment (vishalkarve15) [inline bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryDirectDataWriterHelper.java:178]: Could we just add 2 here? In messageSize?

**Review by vishalkarve15 — APPROVED** [review]

File: bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryDirectDataWriterHelper.java (hunk 1 above)
Comment (agrawal-siddharth) [reply bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryDirectDataWriterHelper.java]: Done

**Pull request merged**